### DISCUSSION TITLE
how to enable JWT Tokens?

### DISCUSSION THREAD

**Discussion by tirelibirefe** [open-metadata/OpenMetadata#12131] [Q&A]
Hello,
I have penMetadata 1.0.4 installed on my EKS/Kubernetes 1.27.
I would like to enable SSO with Keycloak but there are some issues not clear on documentations.

Enable JWT Tokens
There is a sentence [here](https://docs.open-metadata.org/v1.0.0/deployment/security/enable-jwt-tokens#configure-openmetadata-server) which says "Generate UUID64 id to configure JWT_KEY_ID. "

Ok, I want to generate one but it's not documented how to generate it? Could you please advise how to to generate it?

Thanks & Regards

**Comment by harshach** [comment]
@tirelibirefe you can use tool like this https://www.uuidgenerator.net
